shared-components/tl-tournament-card/tl-tournament-card-css.js

```diff
@@ -0,0 +1,79 @@
+import { css } from 'lit';
+
+export const tournamentCardStyles = css`
+    .details-button {
+        justify-content: center;
+        align-items: center;
+        width: 10em;
+        min-height: 4em;
+    }
+    .image-date-wrapper {
+        position: relative;
+    }
+    #date-time-card {
+        position: absolute;
+        top: 0;
+        right: 0;
+        margin: 0 3.6em;
```

Comment:
Why don't you set `right: 3.6rem;` instead of setting `margin`?

Reply:
I wanted to set the same distance on left and right if we have a situation where date card can be centered. Of course, this doesn't need to be the case, but for me it is nicer if it is centered for smaller card. (This depends on design for small screen). That is the only reason.

Reply:
I understand your approach. IMHO, it just simpler to use one setting instead of two, especially when the symetric (you set both sides) has no impact. If it's a concern for the mobile view, I would prefer to have that settings under a media query.